README.md

```diff
@@ -252,6 +252,12 @@ Gradle can generate reports in JUnit XML format. But such reports cannot be used
 
 ## Unreleased
 
+## 0.5.0 (4 October 2023)
```

Comment:
Can you remove the date? I will add it after release.

Reply:
Sure - removed.